server/controllers/requestCtrl.ts

```diff
@@ -60,9 +62,9 @@ const requestCtrl = {
 					{ "request.weekly_budget": { $gte: req.body.budgetLow } },
 					{ "request.weekly_budget": { $lte: req.body.budgetHigh } },
 				],
-			}).select("request");
+			}).select("request");			
 
-			return res.status(204).json(request);
+			return res.status(200).json(res);
```

Comment:
is this meant to return res?

Reply:
My bad. Will change that back to request